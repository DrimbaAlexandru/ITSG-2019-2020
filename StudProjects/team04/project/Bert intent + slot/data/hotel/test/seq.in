i must go to london in 23 august
bye
i will spend less than 300 dollars in berlin
hello
i would like to visit Paris
what can you find in Moscow
many thanks!
find me a hotel in Spain
