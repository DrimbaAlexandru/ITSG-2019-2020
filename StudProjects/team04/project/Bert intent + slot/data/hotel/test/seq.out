O O O O B-toloc.city_name O B-arrive_date.day_number B-arrive_date.month_name
O 
O O O B-money.cost_relative O B-money.price I-money.price O B-toloc.city_name
O
O O O O O B-toloc.city_name
O O O O O B-toloc.city_name
O O
O O O B-hotel O B-toloc.state_name